docs/.vitepress/config.js

```diff
@@ -37,6 +37,17 @@ export default ({ mode }) => {
         "window.dataLayer = window.dataLayer || [];\nfunction gtag(){dataLayer.push(arguments);}\ngtag('js', new Date());\ngtag('config', 'G-MNGVF5NCF7');",
       ],
     ],
+    transformPageData(pageData) {
```

Comment:
The kicker of my issues: You can have a hash where a function is a key and value at the same time 🤯

Reply:
Yep, this is shorthand for `transformPageData: transformPageData(pageData) {`. It can be done with objects too.